tests/integration/mixer/outboundtrafficpolicy/helper.go

```diff
@@ -346,23 +322,45 @@ func setupEcho(t *testing.T, ctx resource.Context, mode TrafficPolicy) (echo.Ins
 			Galley:    g,
 			Ports: []echo.Port{
 				{
+					// Plain HTTP port, will match no listeners and fall through
 					Name:         "http",
 					Protocol:     protocol.HTTP,
-					InstancePort: 8090,
 					ServicePort:  80,
+					InstancePort: 8080,
 				},
 				{
+					// HTTPS port, will match no listeners and fall through
 					Name:         "https",
 					Protocol:     protocol.HTTPS,
-					InstancePort: 8091,
 					ServicePort:  443,
+					InstancePort: 8443,
+					TLS:          true,
 				},
 				{
-					Name:         "tcp",
-					Protocol:     protocol.TCP,
-					InstancePort: 8092,
-					ServicePort:  9090,
+					// HTTPS port, there will be an HTTP service defined on this port that will match
+					Name:        "https-conflict",
+					Protocol:    protocol.HTTPS,
+					ServicePort: 9443,
```

Comment:
I get the re-use/expanding this test but man it's getting hairy to follow all the nuances here. I'm glad at least you augmented it. :)